testing/python/integration.py

```diff
@@ -163,105 +163,6 @@ def test_hello(inject_me):
         reprec = testdir.inline_run()
         reprec.assertoutcome(passed=1)
 
-    def test_unittest_mock_and_pypi_mock(self, testdir):
```

Comment:
that one seems to help track a misbehavior that users might still see

Reply:
Good point, will bring it back then

Reply:
Brought them back, and added `mock` as a testing dependency because of this integration tests.